src/features/zaaksysteem/ZaakDetails.vue

```diff
@@ -72,8 +81,9 @@ const submit = async () => {
   formIsLoading.value = true;
 
   updateToelichting(props.zaak, toelichtingInputValue.value)
-    .then(() => {
+    .then((res) => {
       toast({ text: "De notitie is opgeslagen." });
+      useContactmomentStore().addZaak(res);
```

Comment:
The useContactmomentStore invocation can be moved outside of the function scope:
`const contactmomentStore = useContactmomentStore()`

Reply:
👍🏼